src/components/TextInput/TextInput.module.scss

```diff
@@ -6,7 +6,7 @@
 */
 
 .TextInput {
-  /* Same as Type.scss Body. Should we use a mixin? */
+  /* Same as TypeBase.scss Body. Should we use a mixin? */
   $font-size: 19;
```

Comment:
Not your fault here, but we should probably have a `var(--font-base-size)` or something for all this

Reply:
I'll see if I can work something in.

Reply:
Will address in later PR. comment: 2f69f67